sa/saro.go

```diff
@@ -99,6 +114,12 @@ func (ssa *SQLStorageAuthorityRO) GetRegistration(ctx context.Context, req *sapb
 		// replica yet. If we get a NoRows, wait a little bit and retry, once.
 		ssa.clk.Sleep(ssa.lagFactor)
 		model, err = selectRegistration(ssa.dbReadOnlyMap.WithContext(ctx), query, req.Id)
+		if err != nil {
+			if db.IsNoRows(err) {
```

Comment:
In my two comments, either pick (a) in both or pick (b) in both.

a) Collapse these conditionals into a single line conjoined by `&&`; or
b) Leave as-is